src/main/java/com/redhat/devtools/lsp4ij/server/ProcessStreamConnectionProvider.java

```diff
@@ -77,6 +74,7 @@ public void ensureIsAlive() throws CannotStartProcessException {
 
     protected ProcessBuilder createProcessBuilder() {
         ProcessBuilder builder = new ProcessBuilder(getCommands());
+        builder.environment().putAll(EnvironmentUtil.getEnvironmentMap());
```

Comment:
Do you think it should be done for language server from extension (like Quarkus,Qute LS) ?

Reply:
yes